vision-quickstart/app/src/main/java/com/google/mlkit/vision/demo/VisionProcessorBase.java

```diff
@@ -80,6 +83,8 @@
     private ByteBuffer processingImage;
     @GuardedBy("this")
     private FrameMetadata processingMetaData;
+    private ArrayList<Pair<Image, Long>> bufferImage = new ArrayList<Pair<Image, Long>>();
```

Comment:
OK for experimentation, but if this becomes part of the app, please define an inner class rather than a Pair so you can name the fields